homeassistant/components/knx/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "KNX",
   "documentation": "https://www.home-assistant.io/integrations/knx",
   "requirements": ["xknx==0.14.2"],
-  "codeowners": ["@Julius2342", "@farmio", "@marvin-w"]
+  "codeowners": ["@Julius2342", "@farmio", "@marvin-w"],
+  "quality_scale": "silver"
```

Comment:
Is this done by HA once approved or is it fine that I added this manually?

Reply:
Please list the requirements bullets for the silver level and add a comment  for each.

https://developers.home-assistant.io/docs/integration_quality_scale_index#silver-%F0%9F%A5%88

Reply:
Thank you! I added the bullet points + comments.